stdlib/public/Differentiation_Future/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+set(SOURCES_FOLDER ../Differentiation)
```

Comment:
This seems in conflict with the comment in the parent CMakeLists, that this code should not be shared.

Reply:
It is indeed confusing -- my concern was around the CMake code, but the comment is not clear in this regard

I will sit a bit on the PR to see if I can improve the wording and/or the overall readability of the code

Reply:
I went ahead and did some rewording -- for the sake of clarify, did that in a new PR, https://github.com/apple/swift/pull/34116